loadflow-validation/src/main/java/eu/itesla_project/loadflow/validation/Networks.java

```diff
@@ -0,0 +1,266 @@
+        double p2_calc = rho2 * rho1 * u2 * u1 * y * Math.sin(theta2 - theta1 - ksi + alpha2 - alpha1) + rho2 * rho2 * u2 * u2 * (y * Math.sin(ksi) + g2);
+        double q2_calc = - rho2 * rho1 * u2 * u1 * y * Math.cos(theta2 - theta1 - ksi + alpha2 - alpha1) + rho2 * rho2 * u2 * u2 * (y * Math.cos(ksi) - b2);
+
+        formatter.writeCell(id).writeCell("network_p1").writeCell(p1)
+                 .writeCell(id).writeCell("expected_p1").writeCell(p1_calc)
+                 .writeCell(id).writeCell("network_q1").writeCell(q1)
+                 .writeCell(id).writeCell("expected_q1").writeCell(q1_calc)
+                 .writeCell(id).writeCell("network_p2").writeCell(p2)
+                 .writeCell(id).writeCell("expected_p2").writeCell(p2_calc)
+                 .writeCell(id).writeCell("network_q2").writeCell(q2)
+                 .writeCell(id).writeCell("expected_q2").writeCell(q2_calc);
+
+        if (verbose) {
+            formatter.writeCell(id).writeCell("r").writeCell(r)
+                     .writeCell(id).writeCell("x").writeCell(x)
+                     .writeCell(id).writeCell("g1").writeCell(g1)
+                     .writeCell(id).writeCell("g2").writeCell(g2)
+                     .writeCell(id).writeCell("b1").writeCell(b1)
+                     .writeCell(id).writeCell("b2").writeCell(b2)
+                     .writeCell(id).writeCell("rh01").writeCell(rho1)
+                     .writeCell(id).writeCell("rho2").writeCell(rho2)
+                     .writeCell(id).writeCell("alpha1").writeCell(alpha1)
+                     .writeCell(id).writeCell("alpha2").writeCell(alpha2)
+                     .writeCell(id).writeCell("u1").writeCell(u1)
+                     .writeCell(id).writeCell("u2").writeCell(u2)
+                     .writeCell(id).writeCell("theta1").writeCell(theta1)
+                     .writeCell(id).writeCell("theta2").writeCell(theta2)
+                     .writeCell(id).writeCell("z").writeCell(z)
+                     .writeCell(id).writeCell("y").writeCell(y)
+                     .writeCell(id).writeCell("ksi").writeCell(ksi);
+        }
+        boolean ok = true;
+        if (Math.abs(p1 - p1_calc) > threshold) {
+            LOGGER.warn(id + " P1 " + p1 + " " + p1_calc);
+            ok = false;
+        }
+        if (Math.abs(q1 - q1_calc) > threshold) {
+            LOGGER.warn(id + " Q1 " + q1 + " " + q1_calc);
+            ok = false;
+        }
+        if (Math.abs(p2 - p2_calc) > threshold) {
+            LOGGER.warn(id + " P2 " + p2 + " " + p2_calc);
+            ok = false;
+        }
+        if (Math.abs(q2 - q2_calc) > threshold) {
+            LOGGER.warn(id + " Q2 " + q2 + " " + q2_calc);
+            ok = false;
+        }
+        return ok;
+    }
+    
+    public static boolean checkFlows(Line l, CheckFlowsConfig config) throws IOException, InstantiationException, IllegalAccessException {
```

Comment:
This method was public in the original Networks class. I kept it, just changed for providing the config as input